scriptsConverter.js

```diff
@@ -0,0 +1,44 @@
+document.getElementById("pageConverter").style.display = 'none';
+const result = document.getElementById("result");
+const exchangeRate = document.getElementById("exchangeRate");
+
+function showPageConverter() {
+    const element = document.getElementById("pageConverter");
+    element.style.display = "block";
+    
+}
+
+function getFromCurrency() {
+   return document.getElementById("selectFromValue");
+}
```

Comment:
dałbym tutaj 
```
function getFromCurrencyValue() {
   return document.getElementById("selectFromValue").Value;
}
```
i potem niżej `let fromCurrency = getFromCurrencyValue()` zamiast `let fromCurrency = getFromCurrency().Value `

to co zrobiłeś byłoby super gdybyś wykorzystywał jeszcze inne właściwości inputa selectFromValue. A tak pobierasz tylko wartość więc można od razu to do funkcji wrzucić